# Closing bug #509457 triggered by libwww-perl 5.813 to 5.820
# transition (thanks to Niko Tyni for the patch)
Index: libapache2-mod-perl2/Apache-Test/lib/Apache/TestRequest.pm
===================================================================
--- libapache2-mod-perl2.orig/Apache-Test/lib/Apache/TestRequest.pm	2008-12-26 19:31:56.000000000 -0600
+++ libapache2-mod-perl2/Apache-Test/lib/Apache/TestRequest.pm	2008-12-26 19:32:27.000000000 -0600
@@ -256,6 +256,11 @@
     $self;
 }
 
+sub credentials {
+    my $self = shift;
+    return $self->get_basic_credentials(@_);
+}
+
 sub get_basic_credentials {
     my($self, $realm, $uri, $proxy) = @_;
 
